soccer/Robot.hpp

```diff
@@ -112,7 +114,7 @@ class Robot {
     Context* _context;
 
 private:
-    const unsigned int _shell;
+    const int _shell;
```

Comment:
Why was this reverted to an int instead of unsigned?

Reply:
I guess the main reason is that even though it's used as an `int` in so many places that I thought it would be better to just have it as an `int` instead of an `unigned int`. I can revert this if you like

Reply:
That's fair, it's not that important